### ISSUE TITLE
Pass props to custom component

### ISSUE THREAD

**Issue by loukkyy** [szboynono/mosha-vue-toastify#5]
Great project! Thanks!
Using Vue3 + Vite, can I pass props to my custom component used for the toast?

**Comment by szboynono** [comment]
Hi, thanks for the feedback! As of v1.0.14, you can with props into the custom component with the `withProps` function.

example: 
```ts
    // with props
    import { createToast, withProps } from 'mosha-vue-toastify';
    import CustomizedContent from "./CustomizedContent.vue";
    import 'mosha-vue-toastify/dist/style.css';

    export default defineComponent({
      setup () {
        const toast = () => {
            createToast(withProps(CustomizedContent, { yourFavProp: 'bruh' }))
        }
        return { toast }
      }
    })
  ``` 

Great feedback, thanks again!

**Comment by loukkyy** [comment]
Amazing! 
Just bump my version to `1.0.15` but seems `withProps` is not exposed:
```
Uncaught SyntaxError: The requested module '/node_modules/.vite/mosha-vue-toastify.js?v=fe23573f' does not provide an export named 'withProps'
```

**Comment by szboynono** [comment]
Thanks again!.. That was my bad, something went wrong in the pipeline so the latest build wasn't published, can you try version 1.0.17? it should be fine now

**Comment by loukkyy** [comment]
It works! Thanks for this  amazing component!
<img width="864" alt="Screenshot 2021-07-13 at 23 23 43" src="https://user-images.githubusercontent.com/41201608/125527192-899cb7c4-c946-4dc5-a8f0-4bcd64357a65.png">
